src/pages/VolunteersAbout.page

```diff
@@ -55,7 +55,7 @@
             <div class="slds-col slds-size--1-of-6"  id="colStep3">
                 <div class="slds-text-align--center slds-p-bottom--small">{!$Label.labelVolunteersHelpStep3}</div>
                 <figure class="slds-image">
-                    <a href="https://powerofus.force.com/V4S_Group" class="slds-align--absolute-center">
+                    <a href="https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHiiSAE?tab=discussion&sort=LAST_MODIFIED_DATE_DESC" class="slds-align--absolute-center">
```

Comment:
Should we also update the image? We could maybe just anchor the Trailhead image?
<img width="429" alt="Screen Shot 2022-09-21 at 1 09 55 PM" src="https://user-images.githubusercontent.com/450473/191579318-ba8bf60a-529c-44a0-8c37-d9141c5ac43e.png">

Reply:
Yea, we have to do something about this image.  I was going to demo this on the docs call this afternoon.

Reply:
ok cool, links work. I will test again to make sure the image looks a'ok!